src/CoreShop/Bundle/IndexBundle/Worker/OpenSearchWorker.php

```diff
@@ -198,6 +198,17 @@ public function updateIndex(IndexInterface $index, IndexableInterface $object):
 
         // Delete the document from the index if it is not indexable
         if (!$object->getIndexable($index)) {
+            $documentExists = $client
+                ->exists([
```

Comment:
why not just try catch to avoid the second round-trip?

Reply:
@dpfaffenbauer You're totally right – I changed it.